Report/sections/2_design/Parse_generators.tex

```diff
@@ -0,0 +1,8 @@
+\section{Parse Generators}
+Writing every part of the compiler by hand can be a lenghty process, therefore a tool has been used for fast and effective implemnation of the compiler's scanner and parser. When using a tool it is important to have a good understanding of the choosen generator. The quick implemnation allows us to test the grammar early in the designing phase.
```

Comment:
```suggestion
Writing every part of the compiler by hand can be a lengthy process. A tool has therefore been used to implement the scanner and the parser to shorten this process, while still being just as effective compared to writing it by hand. When using a tool it is important to have a good understanding of the choosen generator. The quick implementation allows us to test the grammar early in the designing phase.
```

Reply:
Sidste sætning bruges der forresten "us". Tror det var i den sammenhæng Gigi sagde vi skulle undgå.
kan ændres til "The qucik implementation makes it possible to test the...", hvis det er.